### DISCUSSION TITLE
Discrepancy with published value for "L'M'S' to IPT" matrix.

### DISCUSSION THREAD

**Discussion by sobotka** [colour-science/colour#705] [General]
In Fairchild's [Brightness, Lightness, and Specifying Color in High-Dynamic-Range Scenes and Images](https://pdfs.semanticscholar.org/1c83/520be9486ebdcdfa597108e983bca16f3f2f.pdf)[PDF] page 9, it lists the L'M'S' to IPT matrix as having a different value as compared with https://github.com/colour-science/colour/blob/master/colour/models/ipt.py#L65

The first value listed in Colour is 4.4550, while in the aforementioned paper, the value is 0.4550

Which version is correct?

**Comment by thomasmansencal** [comment]
The matrix used in Colour is as it appears in [1] and [2]:

![image](https://user-images.githubusercontent.com/99779/30532610-499a3e86-9ca9-11e7-9304-3d2f6f71a62d.png)

However further away in [1] and in the paper you linked [3]:

![image](https://user-images.githubusercontent.com/99779/30532697-ce68ef86-9ca9-11e7-9664-67fbe603837e.png)

So I would say this is very well spotted and I will summon Mark :)

References
---
1. Fairchild, M. D. (2013). IPT Colourspace. In Color Appearance Models (3rd ed., pp. 6197–6223). Wiley. ISBN:B00DAYO8E2
2. Fairchild, M. D., & Wyble, D. R. (2010). hdr-CIELAB and hdr-IPT: Simple Models for Describing the Color of High-Dynamic-Range and Wide-Color-Gamut Images. In Proc. of Color and Imaging Conference (pp. 322–326). ISBN:9781629932156
3. Fairchild, M. D., & Chen, P. (n.d.). Brightness , Lightness , and Specifying Color in High-Dynamic-Range Scenes and Images. https://doi.org/10.1117/12.872075

**Comment by thomasmansencal** [comment]
Mark D. Fairchild has confirmed that the matrix Colour uses is correct:

```
[[ 0.4000  0.4000  0.2000]
 [ 4.4550 -4.8510  0.3960]
 [ 0.8056  0.3572 -1.1628]]
```

This one is not:

```
[[ 0.4000  0.4000  0.2000]
 [ 0.4455 -4.8510  0.3960]
 [ 0.8056  0.3572 -1.1628]]
```
